src/app/components_ngrx/infotip/infotip.component.ts

```diff
@@ -15,9 +20,26 @@ import { Component, Input, HostListener } from '@angular/core';
              </span>`
 })
 
-export class InfotipComponent {
-  @Input() infotipText: string;
+export class InfotipComponent implements OnInit{
+  @Input('uuid') set uuidInput(val: string) {
+    this.infotipsStore.subscribe(i => {
```

Comment:
How many subscriptions are you making for the same data source here? Where are you unsubscribing them?

Reply:
Yes it is still wip, I will do that.